src/both/DeltaNeutralLpV3.sol

```diff
@@ -63,24 +63,34 @@ contract DeltaNeutralLpV3 is AccessStrategy {
         aToken.safeApprove(address(lendingPool), type(uint256).max);
         borrowAsset.safeApprove(address(lendingPool), type(uint256).max);
 
-        // To trade usdc/matic
+        // To trade asset/borrowAsset
         asset.safeApprove(address(_router), type(uint256).max);
         borrowAsset.safeApprove(address(_router), type(uint256).max);
 
         // To add liquidity
         asset.safeApprove(address(_lpManager), type(uint256).max);
         borrowAsset.safeApprove(address(_lpManager), type(uint256).max);
+
+        // xyz/eth chainlink feeds has 18 decimals and xyz/usd chainlink feeds has 8 decimals.
+        decimalDiff = (address(asset) == WETH ? 18 : 8) - asset.decimals();
     }
 
     /// @notice Convert `borrowAsset` (e.g. MATIC) to `asset` (e.g. USDC)
-    function _borrowToAsset(uint256 amountB, uint256 clPrice) internal pure returns (uint256 assets) {
-        // The first divisition gets rid of the decimals of wmatic. The second converts dollars to usdc
-        // TODO: make this work for any set of decimals
-        assets = amountB.mulWadDown(clPrice) / 1e2;
+    function _borrowToAsset(uint256 amountB, uint256 clPrice) internal view returns (uint256 assets) {
+        // The first divisition gets rid of the decimals of `borrowAsset`. The second converts dollars to `asset`
+        if (decimalDiff < 0) {
```

Comment:
`decimalDiff` is a `uint` and cannot be less than zero.

Reply:
My bad, don't what I was thinking. New version does not have this issue.